### ISSUE TITLE
Motion detection support

### ISSUE THREAD

**Issue by simonsickle** [g8keeperzuul/wz_mini_hacks-MQTT#3]
There is a command called waitMotion which will wait for a bit of time and then if motion is detected, will return that motion info. 

To support motion via MQTT another daemon could be created which listens for motion and sets the motion on / off

**Comment by simonsickle** [comment]
I'm taking a look more deeply in this now to see if I can hack something together and send upstream!

**Comment by g8keeperzuul** [comment]
Yes, it is possible to have the camera detect motion and announce that via MQTT. I don't use this feature. Instead I have Frigate do all the motion and object detection using a Coral TPU (much more configurable). 
If you choose to add camera-native support for motion detection, just make sure that it can be turned off (via discovery).

**Comment by simonsickle** [comment]
Would you be fine with a line in the MQTT config to disable / enable motion detection? I could default it to `false` to avoid upgrade issues

**Comment by g8keeperzuul** [comment]
Absolutely

**Comment by falaca** [comment]
Any updates on this @simonsickle ?

I wonder if on this camera it's also possible to send a motion snapshot (i.e., a still image) over MQTT? (Dafang-Hacks supports this iirc)
